dbms/src/Parsers/ParserQueryWithOutput.cpp

```diff
@@ -36,8 +36,10 @@ bool ParserQueryWithOutput::parseImpl(Pos & pos, ASTPtr & node, Expected & expec
 
     ParserKeyword s_ast("AST");
     bool explain_ast = false;
+#if 0
```

Comment:
Hides AST explain command

Reply:
How do I turn it on?

Reply:
#if 1

Then you can add AST before select to get it's AST.